modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_form.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyForm < ApplicationRecord
+    belongs_to :power_of_attorney_request,
+               class_name: 'AccreditedRepresentativePortal::PowerOfAttorneyRequest',
+               inverse_of: :form
+
+    has_kms_key
+
+    has_encrypted :data, key: :kms_key, **lockbox_options
+
+    blind_index :city
+    blind_index :state
+    blind_index :zipcode
+
+    # Validations
+    validates :power_of_attorney_request_id, uniqueness: true
+    validates :data_ciphertext, presence: true
+    validates :city_bidx, presence: true, length: { is: 44 }
+    validates :state_bidx, presence: true, length: { is: 44 }
+    validates :zipcode_bidx, presence: true, length: { is: 44 }
```

Comment:
why length?

Reply:
It's the length of the cryptographic hash that results.